.github/workflows/android_build_lint_test.yaml

```diff
@@ -0,0 +1,37 @@
+name: Android Build & lint
```

Comment:
can we add a check to run this only if we modify anything in packages/android-client folder?

Reply:
This is quite fast task compared eg to react-native-e2e test so conditionally inwoke will not make anything faster.

Reply:
But it can be done